src/utils/utils.js

```diff
@@ -304,19 +304,16 @@ export async function getEvmBech32Address(evmAddress){
  * @returns
  */
 export function getRuntimeConfig(runtimeId){
-    let runtimeConfig = {}
     for (let index = 0; index < PARATIME_CONFIG.length; index++) {
         const runtime = PARATIME_CONFIG[index];
         let runtimeIdList = runtime.runtimeIdList
         for (let j = 0; j < runtimeIdList.length; j++) {
             let config = runtimeIdList[j]
             if(runtimeId === config.runtimeId){
-                runtimeConfig = runtime
-                return runtimeConfig
+                return runtime
             }
         }
     }
-    return runtimeConfig
```

Comment:
I guess this is implicit behavior, but it would be cleaner to explicitly write: `return {}` (or `null`)?

Reply:
Yep, and typescript correctly infers the type too. Added `return undefined`.

(the whole `getRuntimeConfig` is bad code tho. Something to improve in wallet v2)